crates/ibc/src/core/ics04_channel/handler/timeout.rs

```diff
@@ -27,18 +25,18 @@ pub struct TimeoutPacketResult {
 /// packet can no longer be executed and to allow the calling module to safely
 /// perform appropriate state transitions.
 pub fn process<Ctx: ChannelReader>(
-    ctx: &Ctx,
+    ctx_a: &Ctx,
     msg: &MsgTimeout,
 ) -> HandlerResult<PacketResult, PacketError> {
     let mut output = HandlerOutput::builder();
 
     let packet = &msg.packet;
```

Comment:
same here, can we remove this variable?

Reply:
[b9eb033](https://github.com/cosmos/ibc-rs/pull/323/commits/b9eb0335bce88634fb00609d1088ce620791aab6)